charts/generic/container-kill/experiment.yaml

```diff
@@ -8,24 +8,26 @@ metadata:
 spec:
   definition:
     permissions:
-      apiGroups:
+      - apiGroups:
         - ""
```

Comment:
indentation is not correct

Reply:
lets also add the `scope: Namespaced` (alt. Cluster)